Dockerfile

```diff
@@ -31,16 +44,15 @@ for pkg in ${PKGS_TO_INSTALL}; do\n\
   }\n\
 done\n\
 yum -y install ${PKGS_TO_INSTALL}\n' >> /tmp/safe_yum.sh \
-&& /bin/bash /tmp/safe_yum.sh \
-&& yum clean all
+  && /bin/bash /tmp/safe_yum.sh \
+  && yum clean all \
+  && rm /tmp/safe_yum.sh
 
 RUN wget https://bootstrap.pypa.io/get-pip.py \
-  && python3.6 get-pip.py
+  && python3.6 get-pip.py \
+  && pip install -U pip pipenv uwsgi \
+  && rm -rf ~/.cache/pip
 
-#
-# This is needed by click: http://click.pocoo.org/5/python3/
-#
-ENV LC_ALL=en_US.UTF-8
-ENV LANG=en_US.UTF-8
+LABEL version="0.15.9"
```

Comment:
I think even this can go above `yum` command right after ENV declarations. Because, version string is not going to change on every commit or updates in dependencies, so this layer would be wasted each time. However, it is important to keep it independent form other metadata labels.

Reply:
Sounds good. Fixed in f491c19